src/notifications/notification.spec.ts

```diff
@@ -0,0 +1,120 @@
+import {it, describe, expect, injectAsync, TestComponentBuilder} from 'angular2/testing';
+import {Component} from 'angular2/core';
+import {NglNotification} from './notification';
+import {NglNotificationClose} from './notification-close';
+import {provideNglConfig} from '../config/config';
+
+describe('`nglNotification`', () => {
+
+  it('should have the proper classes and attributes', testAsync(({fixture, done}) => {
+    fixture.detectChanges();
+
+    const notificationElement = fixture.nativeElement.querySelector('.slds-notify');
+
+    expect(notificationElement.getAttribute('role')).toBe('alert');
+
+    fixture.componentInstance.type = 'toast';
+    fixture.componentInstance.severity = 'error';
+
+    fixture.detectChanges();
+
+    expect(notificationElement).toHaveCssClass('slds-notify--toast');
+    expect(notificationElement).toHaveCssClass('slds-theme--error');
+
+    fixture.componentInstance.type = 'alert';
+    fixture.componentInstance.severity = null;
+
+    fixture.detectChanges();
+
+    expect(notificationElement).toHaveCssClass('slds-notify--alert');
+    expect(notificationElement).not.toHaveCssClass('slds-theme--error');
+
+    const closeButton = fixture.nativeElement.querySelector('button');
```

Comment:
Extract this `querySelector` into a reusable function since you do this multiple times in your tests.